admin/src/templates/users.html

```diff
@@ -25,15 +25,15 @@
         <div class="col-xs-1"></div>
       </div>
 
-      <div class="row selection" ng-repeat="user in users" ng-click="editUser(user)">
+      <div class="row selection" ng-repeat="user in users" ng-click="!user.inactive && editUser(user)" ng-class="{'inactive': user.inactive}" ng-attr-title="{{user.inactive ? 'Deleted user' : ''}}">
```

Comment:
Please make this translatable (we don't actively translate the admin app but we may do one day and it's a good habit to keep regardless)

Reply:
Fixed